src/components/common/StaffPayment.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <section id="staff-payment-section">
+  <section id="staff-payment">
```

Comment:
The previous id followed our naming convention. Why did you change it?

Reply:
This needs to match 
export enum ReviewSummaryFlags {
  'effective-date-time',
  'court-order',
  'certify-section',
  'staff-payment',
  'document-delivery-section'
}

is probably why :)

Reply:
yea as @jordiwes said below this was to match with the ReviewSummaryFlags, would it be better for me to change the flag instead?

Reply:
Consistent naming is always good, but I see nothing consistent in the above list of flags!

If you have to edit this code for any other reason, please rename. Otherwise just leave it (less work).